HelpSource/Classes/Server.schelp

```diff
@@ -623,6 +623,14 @@ an optional instance of link::Classes/Condition:: used for evaluating this.
 argument:: ... args
 one or more valid OSC messages.
 
+method:: latency
+Returns a floating point variable with the specified server latency
+code::
+~myLocalServer=Server.local;
+~myLocalServer.quit.boot
+~myLocalServer.latency
+::
```

Comment:
This example isn't very insightful. We already know the syntax for how to call instance methods, but what does this method *do*? I'd argue that we don't really need an example for this method at all.